plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/ext/form/field/ItemSelector.js

```diff
@@ -110,6 +110,7 @@ Ext.define('NX.ext.form.field.ItemSelector', {
       // so override these methods to prevent them from including
       // any of their values
       submitValue: false,
+      isDirty: Ext.emptyFn,
```

Comment:
ItemSelector consists of two MultiSelect controls. These controls are used to set the value of ItemSelector, but aren’t treated like normal form fields themselves. I override the isDirty function here, since we don’t care if they’re dirty.